contracts/BondedSortitionPool.sol

```diff
@@ -97,20 +103,40 @@ contract BondedSortitionPool is AbstractSortitionPool {
         );
     }
 
+    /// @notice Sets the minimum bondable value required from the operator
+    /// so that it is eligible to be in the pool. The pool should specify
+    /// a reasonable minimum requirement for operators trying to join the pool 
+    /// to prevent griefing group selection.
+    /// @param minimumBondableValue The minimum bondable value required from the
+    /// operator.
+    function setMinimumBondableValue(uint256 minimumBondableValue) public {
+        require(
+            msg.sender == poolParams.owner,
+            "Only owner may update minimum bond value"
+        );
+
+        poolParams.minimumBondableValue = minimumBondableValue;
+    }
+
+    /// @notice Returns the minimum bondable value required from the operator
+    /// so that it is eligible to be in the pool.
+    function getMinimumBondableValue() public view returns (uint256) {
+        return poolParams.minimumBondableValue;
+    }
+
     function initializeSelectionParams(
-        uint256 currentMinimumStake,
+        uint256 minimumStake,
         uint256 bondValue
     ) internal returns (PoolParams memory params) {
         params = poolParams;
 
-        if (params.minimumBondableValue != bondValue) {
-            params.minimumBondableValue = bondValue;
-            poolParams.minimumBondableValue = bondValue;
+        if (params.requestedBond != bondValue) {
+            params.requestedBond = bondValue;
         }
 
-        if (params.minimumStake != currentMinimumStake) {
-            params.minimumStake = currentMinimumStake;
-            poolParams.minimumStake = currentMinimumStake;
+        if (params.minimumStake != minimumStake) {
+            params.minimumStake = minimumStake;
+            poolParams.minimumStake = minimumStake;
         }
```

Comment:
Do we want to add a check that `requestedBond >= minimumBondableValue`?

Reply:
No, because we may expect someone to have `100 ETH` available (only big fishes allowed in the system) but we still want to allow for `1 ETH`-backed deposits.